packages/svelte-language/package.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "@flint.fyi/svelte-language",
+	"version": "0.0.1",
+	"description": "[Experimental] TypeScript language plugin for Flint.",
```

Comment:
This probably isn't an issue with this PR, but when I think "TypeScript language plugin" I think `compilerOptions.plugins`. "Volar plugin for Svelte"?
I guess long term, we'll want to extract `style` blocks & HTML out as well?